addon/components/initials-avatar.js

```diff
@@ -7,7 +7,21 @@ export default Ember.Component.extend({
   colorIndex: 1,
   maxColorIndex: 1,
 
-  hasImage: Ember.computed.notEmpty('image'),
+  checkImage: function() {
+    var self = this;
+    $.ajax(self.get('image'), {
+      success: function(resp) {
+        self.set('hasImage', true);
+      },
+    });
```

Comment:
This seems problematic. If someone reads hasImage they will get back a value which might flip at some unknown time in the future.